### ISSUE TITLE
Missing _VECT in #pragma interrupt (Renesas RX100 series)

### ISSUE THREAD

**Issue by Plornel** [aws/amazon-freertos#1006]
Illegal #pragma interrupt in port.c l.299 for Renesas RX100 series

Should be : 
```c
#pragma interrupt ( prvTickISR( vect = _VECT( configTICK_VECTOR ), enable ) )
```
Instead of : 
```c
#pragma interrupt ( prvTickISR( vect = configTICK_VECTOR, enable ) )
```

**Comment by abhidixi11** [comment]
Thanks Plornel, we will take a look. I'll post update soon.

**Comment by abhidixi11** [comment]
Hello Plornel,
Did you experience problem with the pragma statement with the latest compiler?  Amazon FreeRTOS does not support RX100 port.c, so I would like make sure. 
Thanks.

**Comment by Plornel** [comment]
Hello abhidixi11,
I encountered no problem with the syntax I suggest.

**Comment by abhidixi11** [comment]
Hello Plornel,

Thanks for the response. I was wondering if you experience any problem with the existing pragma statement ?  (before addition of _VEC).
Thanks.

**Comment by Plornel** [comment]
With the existing statement I have an error "Illegal #pragma interrupt declaration" (toolchain CCRX 3.01.00)

**Comment by yuhui-zheng** [comment]
@Plornel Thank you. We are discussing internally for the other potential impact by merging this change. Shall be able to either merge directly from our side or get back to you with reasoning. Thanks again for bring it up.

**Comment by RichardBarry** [comment]
The important thing here is the compiler version - the syntax needs to be correct for the latest compiler - which compiler version are you using?

**Comment by alexa-noxon** [comment]
@Plornel This has been resolved by #1013.  I will resolve the ticket.  Thank you again for pointing this out!
